app/helpers/govuk_form_helper.rb

```diff
@@ -0,0 +1,60 @@
+module GovukFormHelper
+  class GovukFormBuilder < ActionView::Helpers::FormBuilder
+    def error_messages(key)
+      object.errors[key].map do |msg|
+        @template.content_tag(:span, class: "govuk-error-message") do
+          @template.content_tag(:span, "Error:", class: "govuk-visually-hidden") + @template.sanitize(msg)
```

Comment:
probably should't have nested spans, mozilla says it's not compliant: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span

Reply:
what about this?
https://design-system.service.gov.uk/components/text-input#error-messages

Reply:
I trust mozilla more than GDS

Reply:
oh no, it is permitted, never mind: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content

Reply:
`<marquee>`